src/wasm-lib/kcl/src/lib.rs

```diff
@@ -67,7 +67,7 @@ mod log;
 mod lsp;
 mod modules;
 mod parsing;
-mod settings;
+pub mod settings;
```

Comment:
i dont think this does anything

Reply:
This was required before this [fix](https://github.com/KittyCAD/modeling-app/pull/5563). I will remove this.

When I casted the `Value` to the `Configuration` I needed access to that struct via the settings here.